tools/run_tests/task_runner.py

```diff
@@ -73,6 +73,11 @@ def _create_build_map():
                   default=False,
                   action='store_const',
                   const=True)
+argp.add_argument('-x',
+                  '--xml_report',
+                  default='report_taskrunner_sponge_log.xml',
+                  type=str,
+                  help='Filename for the JUnit-compatible XML report')
```

Comment:
Maybe we need to use the `xml_report` somewhere?

https://github.com/grpc/grpc/blob/dd54a74fefb2065549b409862da701c7e68acd3c/tools/run_tests/task_runner.py#L117

Reply:
:-))) yes that's a good point. I was wondering why the reports don't show up in the way I wanted :-).
Good catch.